python/swiftnav/time.pyx

```diff
@@ -16,9 +16,12 @@ cdef class GpsTime:
 
   def __init__(self, **kwargs):
     memset(&self._thisptr, 0, sizeof(gps_time_t))
-    if kwargs:
-      self._thisptr = kwargs
+    self._thisptr.tow = kwargs.pop('tow')
```

Comment:
Errr, what? Are you sure this doesn't work? There are definitely tests that show that this initializer should work. You have to pass a "**"'d dict to the constructor, like `GpsTime(**{'wn': ...})`.